### ISSUE TITLE
Zoé - Multiplexing cell to cell communication - Bacterial Relay for Energy Efficient Molecular Communications

### ISSUE THREAD

**Issue by Zoepin** [iGEM-Bettencourt-2021/Wet-Lab#45]
**First observation of cell to cell communication:** 
Due to challenges such as molecular cross-talk, no more than two communication systems have been combined in a single design until now (2020).

On the topic of 'relays' in the sensor-relay-effector circuit, this paper enables a single communication system to transmit two separate intercellular conversations.

# Multiplexing cell to cell communication
https://www.embopress.org/doi/full/10.15252/msb.20209618

Channel selectors (CSs):
In electronic engineering, single communication resource such as a wire to transmit multiple conversations.
=> The idea is to implement CSs in cell to cell communication.

### Circuit description
1. Engineer a library of non-cross-reacting (orthogonal) transcriptional NOT and NOR gates. These are based upon promoters with synthetic operator sequences and complementary single guide RNAs.
=> They obtain a circuit in which each gate is affected by a single other gate and can only affect one different element. The RNAs enable a strong repression of the promoters via nuclease-dead Cas9.

_They chose NOT and NOR gates because these gates can be combined to achieve any digital logic operation and readily constructed in live cells using transcriptional repressors and repressible promoters._

2. Those gates are assembled into 2-input MUX and 2-output DEMUX circuits in separate bacteria strains.
=> With the engineered gates, 2 input induces will give a single output compound, here AHL (multiplexing). The AHL, going through different gates, will generate 2 outputs (demultiplexing).

### Design of CS logic circuits and fine tuning of the results
This part is interesting to understand how to design and construct this type of circuits.

### Results
This circuit could be scaled up for more than two cell-cell communication channels.
The circuit requires multiple hours to perform its computation, largely due to the slow decay dynamics of dCas9.

## What this paper means for us
The circuit elegant and effective.
However, it is complicated to design and fine tune. It involves good comprehensive of electronics principles. 
It would be wise to not have to do too many of this (or not at all) in our project.

**Comment by Zoepin** [comment]
This paper tackles bacterial communication efficiency from the point of view physics, electronics, mathematics, mechanics...
The upside is it gives a different approach on biology and cell to cell communication.
The downside is that there are a lot of maths and info it would have taken me ages to understand.

# Bacterial Relay for Energy Efficient Molecular Communications
https://ieeexplore.ieee.org/abstract/document/8013131

It is actually nor very relevant but the maths might be useful at some point, who know???

**Comment by nathaliaraquelx** [comment]
I agree that working with logic gates may be a difficult task, especially the NOT or AND ones...but maybe OR/NOR could be more feasible? And there are some really cool softwares or predicting the behavior of a specific genetic circuit, [here's a paper about one of those - Cello](https://science.sciencemag.org/content/352/6281/aac7341).
